tests/reftests/json.unix.test

```diff
@@ -730,3 +730,436 @@ Done.
     }
   ]
 }
+### : tree :
+### opam switch create tree --empty
+### opam tree --json=out.json
+[ERROR] No package is installed
+# Return code 5 #
+### json-cat out.json
+{
+  "opam-version": "currentv",
+  "command-line": [
+    "${OPAM}",
+    "tree",
+    "--json=out.json"
+  ]
```

Comment:
Should we add here that there was no output (the error ?)

Reply:
I suppose having `null` or having no result (= `undefined`) is enough? The current implementation does not add a result, which means it would be `undefined` if read from JS.